__tests__/database/migration/file/migration_version_2.test.ts

```diff
@@ -0,0 +1,42 @@
+import MigrationVersion2 from '../../../../src/database/migration/file/migration_version_2';
+import {mockDb} from '../../../../__utils__/mockedVariable/mockDb';
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('TESTING Migration Version 2', () => {
+  it('TEST migration up function', async () => {
+    // Setup
+
+    // Execution
+    MigrationVersion2.up(mockDb);
+
+    // Assertion
+    expect(mockDb.executeSql).toHaveBeenCalledWith(
+      `CREATE TABLE IF NOT EXISTS users (
+      id UUID PRIMARY KEY NOT NULL,
+      channel_id UUID NOT NULL,
+      user_id UUID NOT NULL,
+      username TEXT NOT NULL,
+      country_code TEXT,
+      created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+      updated_at DATETIME,
+      last_active_at DATETIME,
+      profile_picture TEXT,
+      bio TEXT,
+      is_banned INTEGER NOT NULL DEFAULT 0
+    );`
+    );
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `up` function of the migration is not being awaited, which could lead to unexpected behavior if the database operation is not completed before the test assertions are run. This could potentially cause flaky tests. Please ensure that you await the promise returned by the `up` function.
```diff
-    MigrationVersion2.up(mockDb);
+    await MigrationVersion2.up(mockDb);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->